migtests/scripts/functions.sh

```diff
@@ -268,3 +268,15 @@ import_data_file() {
 			exit 1
 		}
 }
+
+print_log_file() {
+	log_file_name=$1
+	if [ -f "${EXPORT_DIR}/logs/${log_file_name}" ] && [ ! -f "${EXPORT_DIR}/metainfo/flags/exportDataDone" ]
```

Comment:
Take the `exportDataDone` check outside of this function (consider a case where we want to output voyager log when `export schema` command fails--there won't be this file).

Reply:
Fixed. Btw we havent added an option to tail log files if export schema fails. Should I add?